modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -8483,7 +8483,7 @@ export const console: ConsoleNS = {
                 title: "Roles"
             },
             rolesEdit: {
-                backButton: "Go back to {{type}}",
+                backButton: "Go back to Groups",
```

Comment:
Why was the placeholder removed here?

Reply:
Keep the placeholder and change the passing value for the placeholder (to title case) in the places where this message key is used.

Reply:
Changes made accordingly